### PR TITLE
Feat :new CTA section

### PR DESCRIPTION
**Pull request by cp-mansi-d**
**Tested on:**
- [x] Inspector (all iPhones and android devices like samsung, motorola)
- [x] Desktop (chrome, firefox, safari)
- [ ] Laptop / Macbook  (chrome, firefox, safari)
- [x] iPhone(safari)
- [x] Available android devices  (chrome, firefox)
- [x] iPad(safari)
- [x] Tablet (chrome, firefox)

### CODE REVIEW

**Review by cp-sumi-k — CHANGES_REQUESTED** [review]

File: vue-frontend/tailwind.config.cjs [hunk 1]
```diff
@@ -95,12 +95,17 @@ module.exports = {
...
+          "0%": { transform: "scaleX(0)" },
+          "100%": { transform: "scaleX(1)" },
+        },
       },
       animation: {
         zoomIn: "zoomIn 4s ease-in infinite",
         zoomOut: "zoomOut 4s ease-in infinite",
         typingErase: "typingErase 4s steps(40, end) infinite",
         menuSticky: "0.6s ease-in-out",
+        underlineOut: "underlineOut 1s",
```
Comment (cp-priyanka-g) [inline vue-frontend/tailwind.config.cjs:108]: underline animation looks differ then figma.

File: vue-frontend/src/components/home/NewCTASection.vue [hunk 2]
```diff
@@ -0,0 +1,48 @@
+<template>
+  <div class="tw-relative tw-top-10 sm:tw-top-px tw-pt-12 tw-font-inter-medium">
+    <div class="tw-flex tw-flex-col tw-items-center">
+      <img
```
Comment (cp-priyanka-g) [inline vue-frontend/src/components/home/NewCTASection.vue:4]: background image looks blurr.

File: vue-frontend/src/components/home/NewCTASection.vue [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+        :src="Cta2400"
+        :srcset="`${Cta400} 400w, ${Cta800} 800w, ${Cta1200} 1200w, ${Cta2400} 2400w`"
```
Comment (cp-jagruti-a) [inline vue-frontend/src/components/home/NewCTASection.vue:6]: background image is blur and its circles shapes are getting change in most of devices which is not looking good

File: vue-frontend/src/components/home/NewCTASection.vue [hunk 4]
```diff
@@ -0,0 +1,48 @@
+<template>
+  <div class="tw-relative tw-top-10 sm:tw-top-px tw-pt-12 tw-font-inter-medium">
```
Comment (cp-priyanka-g) [inline vue-frontend/src/components/home/NewCTASection.vue:2]: recheck the font and font-weight

**Review by cp-sumi-k — DISMISSED** [review]

**Review by cp-priyanka-g — APPROVED** [review]

**Review by cp-jagruti-a — APPROVED** [review]

**Review by cp-sumi-k — APPROVED** [review]

**Pull request merged**